internal/api/api.go

```diff
@@ -85,6 +86,9 @@ func NewAPIWithVersion(ctx context.Context, globalConfig *conf.GlobalConfigurati
 		r.UseBypass(api.databaseCleanup)
 	}
 
+	if globalConfig.Logging.Level == "debug" {
+		r.chi.Mount("/debug", chimiddleware.Profiler())
+	}
```

Comment:
I feel like this should be another HTTP server?

Reply:
i don't think it's possible to profile the main server while having pprof run on a separate server? but since this leaks important runtime info, i think we can put it behind the admin JWT - wdyt?